Enable TTS for prebuilt macOS C-API

Can you enable TTS for macOS C API?

```console
sherpa-onnx-v1.10.36-osx-universal2-shared.tar.bz2	01b64450ad10b5c9531068c99a6411dcc5709b5de4fcaabb0e510d2b3d71a162
sherpa-onnx-v1.10.36-osx-universal2-static.tar.bz2	ea5d16f048686d4b4c728d49a4aac1f9f7a2894856fbd183c8be7a8ae9e8f355
```

These files doesn't contain it.
I think this needs to be enabled here:
https://github.com/k2-fsa/sherpa-onnx/blob/08d771337baad95717e1ebb68a468568f2505fe2/build-swift-macos.sh#L21C5-L21C29

My error:

```console
Undefined symbols for architecture arm64:
            "_SherpaOnnxCreateOfflineTts", referenced from:
                sherpa_rs::tts::OfflineTts::new::ha1b3f2adf3abda92 in libsherpa_rs-0a21562d903a3ce0.rlib[84](sherpa_rs-0a21562d903a3ce0.cur3x6enpr8149uaur1xongvr.rcgu.o)
```

--- a/.github/workflows/macos.yaml
+++ b/.github/workflows/macos.yaml
@@ -269,11 +269,12 @@ jobs:
           .github/scripts/test-online-transducer.sh
 
       - name: Copy files
+        if: matrix.build_type == 'Release'
         shell: bash
         run: |
           SHERPA_ONNX_VERSION=v$(grep "SHERPA_ONNX_VERSION" ./CMakeLists.txt  | cut -d " " -f 2  | cut -d '"' -f 2)
 
-          if [[ ${{ matrix.with_tts }} ]]; then
+          if [[ ${{ matrix.with_tts }} == ON ]]; then
             dst=sherpa-onnx-${SHERPA_ONNX_VERSION}-osx-universal2-${{ matrix.lib_type }}
           else
             dst=sherpa-onnx-${SHERPA_ONNX_VERSION}-osx-universal2-${{ matrix.lib_type }}-no-tts
@@ -290,7 +291,7 @@ jobs:
           tar cjvf ${dst}.tar.bz2 $dst
 
       - name: Release pre-compiled binaries and libs for macOS
-        if: (github.repository_owner == 'csukuangfj' || github.repository_owner == 'k2-fsa') && github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+        if: matrix.build_type == 'Release' && (github.repository_owner == 'csukuangfj' || github.repository_owner == 'k2-fsa') && github.event_name == 'push' && contains(github.ref, 'refs/tags/')
         uses: svenstaro/upload-release-action@v2
         with:
           file_glob: true

